clippy_tests/examples/borrow_box.rs

```diff
@@ -0,0 +1,24 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+#![deny(borrowed_box)]
+#![allow(blacklisted_name)]
+#![allow(unused_variables)]
+#![allow(dead_code)]
+
+pub fn test1(foo: &mut Box<bool>) {
+    println!("{:?}", foo)
+}
+
+pub fn test2() {
```

Comment:
Can you add a trait method taking a box reference and an impl to show that only the trait is linted and not the impl?